src/algebra/group/to_additive.lean

```diff
@@ -181,6 +147,105 @@ aux (λ n, (env.constructors_of n).mmap $
                 | _ := fail "Bad constructor name"
                 end)
 
+/--
+The attribute `to_additive` can be used to automatically transport theorems
+and definitions (but not inductive types and structures) from a multiplicative
```

Comment:
Most of the text here was moved from the module doc string, but I also added some new sections and examples.